ql/arch/cc_light/cc_light_eqasm_compiler.h

```diff
@@ -757,14 +757,17 @@ class cc_light_eqasm_compiler : public eqasm_compiler
         // schedule with platform resource constraints
         ql::ir::bundles_t bundles = cc_light_schedule_rc(ckt, platform, num_qubits);
 
-        // write RC scheduled bundles with parallelism as simple QASM file
-        std::stringstream sched_qasm;
-        sched_qasm <<"qubits " << num_qubits << "\n\n"
-                   << ".fused_kernels";
-        string fname( ql::options::get("output_dir") + "/" + prog_name + "_scheduled_rc.qasm");
-        IOUT("Writing Recourse-contraint scheduled CC-Light QASM to " << fname);
-        sched_qasm << ql::ir::qasm(bundles);
-        ql::utils::write_file(fname, sched_qasm.str());
+        if( ql::options::get("write_gasm_files") == "yes")
```

Comment:
I think there is a typo. it should say `write_qasm_files`, right? (same thing with the rest of files)

Reply:
indeed. thanks. updating it in the next commit